docs/source/documentation.html.md.erb

```diff
@@ -59,6 +59,41 @@ As a result, you will get the Algolia Places autocompletion menu:
 
 <input type="search" id="docs-getting-started-demo" class="form-control" placeholder="Where are we going?" />
 
+## Sign up & Rate limits
+
+The Algolia Places API enforces some rate limits.
```

Comment:
`The Algolia Places API is free, but it enforces some rate limits. You get up to 1,000 requests per day without authenticating, 100,000 per month with authenticating. More is also possible, but that will not be free anymore. `

This is then replicated in the table